bangazonapi/views/profile.py

```diff
@@ -81,8 +81,11 @@ def list(self, request):
             }
         """
         try:
-            current_user = Customer.objects.get(user=4)
+            current_user = Customer.objects.get(user=request.auth.user)
```

Comment:
Can you add a note to the pr description that this pr also fixes ticket #22?

Reply:
added those changes and re pushed